### PR TITLE
ICO changes

### PR DESCRIPTION
**Pull request by 4p00rv**

### CODE REVIEW

File: src/javascript/binary/websocket_pages/user/ico_portfolio.js [hunk 1]
```diff
@@ -60,6 +62,7 @@ const ICOPortfolio = (() => {
             .append($('<td/>', { class: 'payout' }).append($('<strong/>', { text: shortcode[2] })))
             .append($('<td/>', { class: 'bid' }).append($('<strong/>', { html: formatMoney(data.currency, shortcode[1]) })))
             .append($('<td/>', { class: 'purchase' }).append($('<strong/>', { html: formatMoney(data.currency, data.buy_price) })))
+            .append($('<td/>', { class: 'deposit' }).append($('<strong/>', { html: formatMoney(data.currency, data.buy_price * deposit_factor) })))
```
Comment (negarn) [inline src/javascript/binary/websocket_pages/user/ico_portfolio.js]: is it worth changing to `+data.buy_price`?

File: src/javascript/binary/websocket_pages/user/ico_portfolio.js [hunk 2]
```diff
@@ -30,9 +30,11 @@ const ICOPortfolio = (() => {
     };
 
     const createPortfolioRow = (data, is_first) => {
-        const long_code  = data.longcode;
-        const ico_status = (State.getResponse('ico_status.ico_status') || '').toLowerCase();
-        // Default to cancel bid. Ended on a button doesn't make sense.
+        const long_code          = data.longcode;
+        const ico_status         = (State.getResponse('ico_status.ico_status') || '').toLowerCase();
+        const deposit_percentage = +((data.shortcode || '').split('_')[2] || State.getResponse('ico_status.initial_deposit_percentage'));
+        const deposit_factor     = (deposit_percentage || 5) / 100;
```
Comment (negarn) [inline src/javascript/binary/websocket_pages/user/ico_portfolio.js]: is it worth changing to `+deposit_percentage`?

File: src/javascript/binary/websocket_pages/user/ico_subscribe.js [hunk 3]
```diff
@@ -102,7 +109,7 @@ const ICOSubscribe = (() => {
                     { selector: '#price',    validations: ['req', ['number', { type: 'float', decimals: `1, ${decimal_places}`, min: Math.pow(10, -decimal_places).toFixed(decimal_places) }]], no_scroll: 1 },
 
                     { request_field: 'buy', value: 1 },
-                    { request_field: 'amount',        parent_node: 'parameters', value: () => document.getElementById('price').value },
+                    { request_field: 'amount',        parent_node: 'parameters', value: () => `${document.getElementById('price').value}/${initial_deposit_percent}` },
```
Comment (negarn) [inline src/javascript/binary/websocket_pages/user/ico_subscribe.js]: same here, is it worth doing `+document.getElementById('price').value` and `+initial_deposit_percent`?

File: src/javascript/binary/websocket_pages/user/ico_subscribe.js [hunk 4]
```diff
@@ -131,54 +138,28 @@ const ICOSubscribe = (() => {
         const price_val    = $price.val();
         let total          = 0;
         let usd_total      = 0;
+        const deposit_factor = initial_deposit_percent/100;
```
Comment (negarn) [inline src/javascript/binary/websocket_pages/user/ico_subscribe.js]: also here

File: src/javascript/binary/websocket_pages/user/ico_portfolio.js (hunk 2 above)
Comment (4p00rv) [reply src/javascript/binary/websocket_pages/user/ico_portfolio.js]: @negarn  Doing it right above the line.

File: src/javascript/binary/websocket_pages/user/ico_subscribe.js (hunk 3 above)
Comment (4p00rv) [reply src/javascript/binary/websocket_pages/user/ico_subscribe.js]: @negarn This is supposed to be a string of format `1.35/5`.

Comment (negarn) [reply src/javascript/binary/websocket_pages/user/ico_subscribe.js]: oops :)

File: src/javascript/binary/websocket_pages/user/ico_portfolio.js (hunk 2 above)
Comment (negarn) [reply src/javascript/binary/websocket_pages/user/ico_portfolio.js]: you're right, thanks :)

**Pull request merged**